<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationMenuSettingsDialog</class>
 <widget class="QDialog" name="ApplicationMenuSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application Menu Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>500</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>241</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fontSizeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>94</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Font size</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundTransparencyLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Background transparency</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontSize">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>63</width>
     <height>44</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="backgroundTransparency">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>210</y>
     <width>81</width>
     <height>44</height>
    </rect>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="iconSizeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>94</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="iconSize">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>63</width>
     <height>44</height>
    </rect>
   </property>
   <property name="minimum">
    <number>32</number>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
   <property name="singleStep">
    <number>8</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ApplicationMenuSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ApplicationMenuSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
